hpi-swt2/lecture-portal
# Feature/#83 Implement and test file upload.

# Resolves #83
This PR introduces the feature of file upload.

Acceptance Criteria:
- [x] A "File" DB-Scheme exists
- [x] In the scheme, a file can be associated with a lecture, or a course


Testing steps:
- go to "/uploaded_files"
- click "Upload file"
- enter a valid lecture id
- select a file
- click "Create Uploaded file"
- you should be redirected to "/uploaded_files" and see the file you just uploaded

Steps needed for migration:
- db migrations

---

Definition Of Done:
